nerfstudio/viewer_beta/viewer.py

```diff
@@ -329,7 +342,7 @@ def update_scene(self, step: int, num_rays_per_batch: Optional[int] = None) -> N
         """
         self.step = step
 
-        if self.camera_state is None:
+        if len(self.render_statemachines) is None:
```

Comment:
Is this condition still doing something? Will this ever be `None`?

Reply:
Oops I changed it now to `if len(self.render_statemachines) == 0`